zds/utils/templatetags/pluralize_fr.py

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+
+from django import template
+
+register = template.Library()
+
+
+@register.filter(is_safe=False)
+def pluralize_fr(value, arg='s'):
+    """
+    Based on default pluralize filter : https://github.com/django/django/blob/master/django/template/defaultfilters.py
+    Support french (-1, 0 and 1 are singular).
+    """
+    if ',' not in arg:
+        arg = ',' + arg
+    bits = arg.split(',')
+    if len(bits) > 2:
+        return ''
+    singular_suffix, plural_suffix = bits[:2]
+
+    try:
+        if float(value) not in [-1, 0, 1]:
```

Comment:
C'est de la récup' du filtre de Django, et ça fonctionne en anglais mais pas en français. Le filtre pour l'anglais, c'est "Pluriel si `float(val) != 1`. C'est correct parce qu'en anglais c'est "1.2 things", pas "1.2 thing". En français la règle est différente, ça doit être "pluriel si `1 <= val < 2`". "1 chose, 1.5 chose, 2 choses".

Donc ici il faudrait plutôt `if not (-2 < float(value) < 2):`